src/state/CallViewModel.ts

```diff
@@ -504,7 +504,9 @@ export class CallViewModel extends ViewModel {
                     (grid, spotlight, screenShares): Layout => ({
                       type: "grid",
                       spotlight:
-                        screenShares.length > 0 ? spotlight : undefined,
+                        screenShares.length > 0 || grid.length > 20
```

Comment:
the 20 seems somewhat random. I think this should either become a config or needs a comment what made us decide to hard code this to 20.

Reply:
maybe also a "needs to show scrollbars" is the better condition?